src/app/work-item/work-item-list/work-item-list.component.scss

```diff
@@ -28,5 +29,28 @@
     @include transition(opacity .3s);
     opacity: .6;
     width: 100%;
-  }
+  }  
+}
+
+.detail-panel {
+  background: $color-pf-white;  
+  position: fixed;
+  left: auto;
+  top: 34px;
+  right: 0;
+  bottom: 0;
+  width: 50%;
+  min-width: 250px;
+  z-index: 9999;
```

Comment:
curious as to why the z-index is set so high. Are there 9998 other layers that this needs to be above?